.ci.yaml

```diff
@@ -156,6 +156,19 @@ targets:
       emulator_arch: "x64"
     timeout: 60
 
+  - name: Linux Fuchsia FEMU (cso)
+    recipe: engine/femu_test
+    bringup: true
+    properties:
+      add_recipes_cq: "true"
+      build_fuchsia: "true"
+      fuchsia_ctl_version: version:0.0.27
```

Comment:
This is not needed, fuchsia_ctl was removed a few months ago.

Reply:
This is a copy paste from the other Fuchsia config. Let's remove in a separate cleanup.